Source/Modules/Documentarian/Documentation/reference/cmdlets/hash2yaml.md

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
I think this file can be deleted since it's a private function. 🤔

Reply:
Fixed